### ISSUE TITLE
[App]: Separating config and metrics

### ISSUE THREAD

**Issue by albertfgu** [wandb/wandb#5446]
### Current Behavior

In the Run Comparer, I would like it to only show the _config_ of the models (e.g. hyperparameters) so that I can compare differences between runs. The Run comparer seems to show every single logged metric, which is counterintuitive since config and metrics are different types of things.

More to the point, this seems to occur consistently in other places which might classify as a bug. In particular, in the run Overview page, the left side shows the _config_ and the right side shows the _metrics_ as appropriate (screenshot)
![image](https://user-images.githubusercontent.com/8164679/235170169-c13b5ccd-b36e-4300-8b46-2b27346b70fa.png)

However, when I tried to download the config as raw dictionary data (pressing the "View Raw Data" button on the left), it exports not only the config but all the metrics again. I think this is a bug, unless it was an intentional design choice for some reason?

### Expected Behavior

Better separation of config and metrics where appropriate
- In the "View Raw Data" exports, the config export should only give the config and the metrics export should only give the metrics
- [feature request] in the Run Comparer, add a flag to only show config or metrics if desired

### Environment

OS: macOS 13.3

Browsers: Chrome

Version:

**Comment by nate-wandb** [comment]
Hi @albertfgu, I can make the feature request and also report the bug to our engineering team. I'm not sure if they showing both metrics and config when you click Raw Data is by design but I agree that it would make sense to break these out and separate them. 

I can follow up once the engineering team has had a chance to look into these.

Thank you,
Nate

**Comment by albertfgu** [comment]
Thanks!

**Comment by nate-wandb** [comment]
Hi @albertfgu, I wanted to follow up on this. I'm not seeing the same behavior when it comes to downloading the raw config file. I see the config that is passed to `wandb.init` along with the additional metadata we collect but I don't see any metric values here. Could you possibly share a link to a run where you are seeing this?

Thank you,
Nate

**Comment by nate-wandb** [comment]
Hi @albertfgu, just wanted to follow up on this and see if you had a run where you were seeing the config and summary combined?

**Comment by albertfgu** [comment]
Hi Nate, sorry for getting back late - I don't have much time to look into it for a few weeks. I think this happens in all my wandb runs, here's an old random run I pulled out: https://wandb.ai/hazy-research/hippo/runs/d66uw9ly/overview?workspace=

Now that I look at it more carefully, though, all the metrics are under a `_wandb` key. Now I'm wondering if this is something automatically added to the config by [Lightning's wandblogger](https://lightning.ai/docs/pytorch/stable/extensions/generated/lightning.pytorch.loggers.WandbLogger.html) which I'm using, although I can't imagine why/how they're getting logged to the config.

**Comment by nate-wandb** [comment]
Hi @albertfgu, Sorry for the delay on this but I've made a feature request to remove these from the config when using PTL's WandbLogger class. I can follow up once the engineering team has a chance to look into this.
